### DISCUSSION TITLE
Giscus integration

### DISCUSSION THREAD

**Discussion by Jak2k** [dailydotdev/daily#1063] [Feature Requests]
Many sites have comments powered by [giscus](https://giscus.app/). It would be great, if daily.dev suported that type of comments. Giscus comments are just answers to a GitHub-Disscussion, so integrating it should be easy: You just have to look, whether the post includes the giscus script and when it has, you have to read the attributes, that tell which repo and disscussion belongs to the post. Then you could show these comments on daily.dev and allow users to comment using it.

This would be good for daily.dev because there get more comments to it and it would be good for creators, because they have all their comments in one place.

**Comment by idoshamun** [comment]
Hi @Jak2k,

Thanks for another suggestion :)
Can you please give me some examples of sources you can find on daily.dev that use giscus?

**Reply by Jak2k** [reply]
https://app.daily.dev/posts/1sx4MAI15

**Reply by idoshamun** [reply]
My concern is that we don't have so many posts with giscus. I never seen it at least, except this one article. This is why I was wondering if there are sources that use it
